test/.eslintrc

```diff
@@ -2,9 +2,10 @@
   "env": {
     "jest/globals": true
   },
-  "plugins": ["jest"],
+  "plugins": ["jest", "shopify"],
```

Comment:
Do we really need to enable this for the tests? Doesn't seem like it would provide much benefit here.

Reply:
Makes sense. We don't really need it here.